interface/forms/eye_mag/help.php

```diff
@@ -35,7 +35,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="description" content="Eye Exam Help" />
     <meta name="author" content="openEMR: ophthalmology help" />
-    <?php Header::setupHeader(['jquery-ui', 'jquery-ui-excite-bike']); ?>
+    <?php Header::setupHeader(); ?>
```

Comment:
Hello @bradymiller and @tywrenn 
I would like to use Bootstrap collapse components. But it seems like Bootstrap js didn't load completely.

![image](https://user-images.githubusercontent.com/22230889/82335917-863b0c80-9a1c-11ea-80c6-eeb63d109eb1.png)

Reply:
JS shouldn't be required to use Bootstrap's accordion collapse components. This JS might need to be looked at again or replaced entirely.

Reply:
@tywrenn  Yup, I found in my codebase has a little bit code conflict that caused this issue. I will clone the new project after I finished #3497